src/model/CMakeLists.txt

```diff
@@ -78,6 +78,54 @@ facelift_add_library(FaceliftModelLib
         StructTypeHandler.h
         PropertyInterface.h
         FaceliftProperty.h
+    HEADERS_NO_MOC
+        ModelProperty.h
```

Comment:
Each header should be defined only once.

Reply:
fixed